test/e2e/testdata/reports/persistentvolumeclaim-usage.json

```diff
@@ -1 +1,121 @@
\ No newline at end of file
+[
```

Comment:
I took a quick look at the commits, but it's difficult to tell what's been changed vs. what's just been pretty-printed.

Reply:
This image should help, it shows the report file which had real value changes. I soft-wrapped the version from master:
<img width="1766" alt="image" src="https://user-images.githubusercontent.com/4449199/83287677-106b3980-a1b0-11ea-9566-91a57e74c981.png">

Reply:
<img width="1626" alt="image" src="https://user-images.githubusercontent.com/4449199/83288083-cafb3c00-a1b0-11ea-9c70-6ce0ad3fa735.png">

Reply:
<img width="1634" alt="image" src="https://user-images.githubusercontent.com/4449199/83288186-f1b97280-a1b0-11ea-8eb3-3d483d9acb94.png">